indra/databases/chebi_client.py

```diff
@@ -148,6 +153,44 @@ def _read_relative_csv(rel_path):
     return csv_reader
 
 
+@lru_cache(maxsize=50)
```

Comment:
Link this function to `get_chebi_name_from_id` so it is called when the ID is not found in the local resource file